src/browser/cacheDisplay.html

```diff
@@ -27,6 +27,52 @@
                         <md-button ng-click="$ctrl.removeCache(0, $event)" ng-disabled="$ctrl.cacheInfo.disableDeleteCache"><i class="material-icons">delete</i></md-button>
```

Comment:
Ok yeah to answer my own question, it seems like moving this line below the md-card-content area you added fixes the issue brought up in my earlier comment.